.vettedpositions

```diff
@@ -317,3 +316,8 @@
 /pkg/util/pubsub/http_handler.go:51:40: requestcontext
 /pkg/util/sentry/request.go:25:9: requestcontext
 /pkg/util/template/engine.go:307:57: requestcontext
+/cmd/portal/cmd/cmdinternal/migrate_resources_migrate_rate_limits.go:65:24: contextbackground
+/pkg/lib/config/configsource/resources.go:105:34: contextbackground
+/pkg/lib/config/configsource/resources.go:504:9: contextbackground
+/pkg/lib/config/configsource/resources.go:670:9: contextbackground
```

Comment:
We should not allow `contextbackground` usage.

Reply:
Fixed, thanks!